account_asset_management/models/account_move.py

```diff
@@ -73,11 +76,15 @@ def post(self):
                 }
                 if self.env.context.get("company_id"):
                     vals["company_id"] = self.env.context["company_id"]
-                asset = (
-                    self.env["account.asset"]
-                    .with_context(create_asset_from_move_line=True, move_id=move.id)
-                    .create(vals)
-                )
+                temp_asset = AccountAsset.new(vals)
```

Comment:
Instead of doing this, I would prefer to use `Form` class to populate the asset, which is very comfortable. I can do it if you want.

Reply:
I prefer using new

Reply:
Using `new`, you are not populating default values.